imap_processing/glows/l0/glows_decom.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+
+from bitstring import ReadError
+from space_packet_parser import parser, xtcedef
+
+from imap_processing import imap_module_directory
+from imap_processing.glows.l0.glows_l0_data import GlowsDeL0, GlowsHistL0
+
+
+def decom_packets(packet_file_path: str) -> tuple[list[GlowsHistL0], list[GlowsDeL0]]:
+    """Decom GLOWS data packets using GLOWS packet definition
+    Parameters
+    ----------
+    packet_file_path : str
+        Path to data packet path with filename
+
+    Returns
+    -------
+    data : tuple[list[GlowsHistL0], list[GlowsDeL0]]
+        A tuple with two pieces: one list of the GLOWS histogram data, in GlowsHistL0
+        instances, and one list of the GLOWS direct event data, in GlowsDeL0 instance
+    """
+    hist_apid = 1480
+    de_apid = 1481
```

Comment:
HIT and Ultra are using Enum/IntEnum for these, do you want to do the same here?